app/workers/document_list_export_worker.rb

```diff
@@ -16,12 +16,10 @@ def perform(filter_options, user_id)
   def send_mail(csv_filename, user, filter)
     MailNotifications.document_list(csv_filename, user.email, filter.page_title).deliver_now
   rescue Notifications::Client::BadRequestError => e
-    if !%w[integration staging].include?(ENV["SENTRY_CURRENT_ENV"]) || e.message !~ /team-only API key/
-      # in production we care about all errors
-      # in staging and integration the team-only error is unrecoverable when running asynchronously
-      # (team-only error is unrecoverable in production too, but almost certainly impossible)
-      raise
-    end
+    raise if ENV["SENTRY_CURRENT_ENV"] =~ /production/ || e.message !~ /team-only API key/
```

Comment:
Nitpick: the `raise` is above the comment in this file, but below it in the other. It should be consistent (below the comment, IMO).

Otherwise looks good - thanks Ed! 👍